README.md

```diff
@@ -95,6 +95,51 @@ api.find(filterFunc).forEach(function (request) {
 });
 ```
 
+#### Writing an Adapter
+
+Adapters convert from an input format such as API Blueprint into refract elements. This allows a single, consistent interface to be used to interact with multiple input API description formats. Writing your own adapter allows you to add support for new input formats.
+
+Adapters are made up of a name, a list of media types, and three public functions: `detect`, `parse`, and an optional `serialize`. A simple example might look like this:
+
+```js
+export const name = 'my-adapter';
+export const mediaTypes = ['text/vnd.my-adapter'];
+
+export function detect(source) {
+  // If no media type is know, then we fall back to auto-detection. Here you
+  // can check the source and see if you think you can parse it.
+  return source.match(/some-test/i) !== null;
+}
+
+export function parse({source, sourceMap}, done) {
+  // Here you convert the source into refract elements.
+  // ...
+  done(null, elements);
+}
+
+export function serialize(api, done) {
+  // Here you convert from javascript objects to the serialized
+  // source format.
+  // ...
+  done(null, outputString);
+}
+```
+
+Now you can register your adapter with Fury.js:
+
+```js
+import * as fury from 'fury';
+import * as myAdapter from './my-adapter';
+
+// Register my custom adapter
+fury.adapters.unshift(myAdapter);
```

Comment:
This is a good point. After thinking about this for a while I think I'm going to modify it so that the default export is an instance of a `Fury` class, so the code above will work the same more or less, but you can also import the class itself (from a named export) and instantiate it yourself with your own adapters as many times as you might need. It gives us the best of both worlds with almost no extra complexity.